packages/main/src/Card.js

```diff
@@ -47,6 +48,18 @@ const metadata = {
 	},
 	properties: /** @lends sap.ui.webcomponents.main.Card.prototype */ {
 
+		/**
+		 * Defines the accessible name of the component.
+		 * <b>Note:</b> <code>accessibleName</code> is required for the component, it provides unique name for the card region.
```

Comment:
How about:
<b>Note:</b><code>accessibleName</code> is used as the name of the card region and should be unique.